puns.json

```diff
@@ -265,5 +265,9 @@
   {
     "pun": "Q: What search algorithm did the wise men use to find the baby Jesus?",
     "punchline": "A: A*"
+  },
+  {
+    "pun": "Q: Why was the JavaScript developer sad?",
+    "punchline": "A: Because he didn't Node how to Express himself!"
```

Comment:
Hey @kamaroolkarim , thanks for the PR.

I'm trying to keep the puns gender-neutral, could you change it from `he` to `they`? Same for `himself` to `themselves`.

Reply:
ok. let me fix this